### PR TITLE
no tcmalloc

### PR DESCRIPTION
**Pull request by josh-cogdill**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -48,7 +48,6 @@ list(APPEND LIBS ${STOPWATCH})
 find_library(DEATHKNELL DeathKnell PATHS /usr/local/probe/lib )
 list(APPEND LIBS ${DEATHKNELL})
 
-find_library(TCMALLOC tcmalloc PATHS /usr/local/probe/lib )
 # Appended first instead
```
Comment (KjellKod) [inline CMakeLists.txt]: @josh-cogdill please remove the comment also

**Comment by KjellKod** [comment]
👍

**Comment by john-gress** [comment]
I think you need to update the README.md file - right?

**Comment by john-gress** [comment]
👍

**Pull request merged**